### DISCUSSION TITLE
Whether to consider the configuration of maximum path layers

### DISCUSSION THREAD

**Discussion by likun7981** [sindresorhus/pure#627] [General]
### General information
I used a single line, and the following happens when it takes too long path
![image](https://user-images.githubusercontent.com/13427467/168888671-485034ba-7542-4597-b641-d7cfb20d3a9a.png)

Whether to consider the configuration of path tier count, for example, I set the maximum display to three layers of path

![image](https://user-images.githubusercontent.com/13427467/168888872-1dcc2634-9f6c-4a21-8fa3-60030a00cf1f.png)

**Comment by sindresorhus** [comment]
> I used a single line, and the following happens when it takes too long path

This is one of the reasons Pure uses two lines.

---

We don't plan to add support for this built-in, but maybe someone here can help you achieve it.

**Reply by likun7981** [reply]
> This is one of the reasons Pure uses two lines.

In fact, sometimes two lines can feel very long.

> We don't plan to add support for this built-in, but maybe someone here can help you achieve it.

If you are considering adding this configuration, I can sumit a pullRequest to achieve it.
